cockatrice/src/chatview.cpp

```diff
@@ -374,9 +417,27 @@ void ChatView::appendMessage(QString message, QString sender, UserLevelFlags use
                 while (fullMentionUpToSpaceOrEnd.size());
             }
         }
+        else if (index == highlightWordFirstIndex)
+        {
+            // You have received a valid mention of custom word!!
+            int firstSpace = (message.indexOf(" ") == -1 ? message.size() : message.indexOf(" "));
+            mentionFormat.setBackground(QBrush(getCustomMentionColor()));
+            mentionFormat.setForeground(settingsCache->getChatMentionForeground() ? QBrush(Qt::white) : QBrush(Qt::black));
+            cursor.insertText(message.mid(0, firstSpace), mentionFormat);
+            cursor.setCharFormat(defaultFormat);
+            message = message.mid(firstSpace);
+            QApplication::alert(this);
+            if (settingsCache->getShowMentionPopup() && shouldShowSystemPopup())
+            {
+                QString ref = sender.left(sender.length() - 2);
+                showSystemPopup(ref);
+            }
```

Comment:
Done, but I left in the alert as I like the icon bouncing/notifying me somebody said something I'd like to be highlighted